src/content/documentation/develop/developing-extensions-for-firefox-for-android.md

```diff
@@ -112,6 +114,18 @@ When setting `strict_min_version` in [`browser_specific_settings`](https://devel
 }
 ```
 
+### Android API and UX checklist
+
+Before making your extension publicly available on AMO, test your extension on Android to make sure it works as expected. Reviewers may reject or request changes to submissions that don't meet basic usability requirements.
```

Comment:
Shouldn't this advice apply regardless of how the developer makes the extension available?
```suggestion
Before making your extension available, test it on Android to ensure it works as expected. When you publish it on AMO, reviewers may reject it or request changes if it doesn't meet basic usability requirements.
```

Reply:
I don't know how much reviewers take usability into account for typical extension submissions. It's an acute concern for mobile extensions because typical desktop-focused development practices lead to especially poor experiences on mobile.